.github/workflows/test.yml

```diff
@@ -3,8 +3,7 @@ name: Tests
 on:
   pull_request:
     branches:
-      - dev
-      - main
+      - *
```

Comment:
The characters *, [, and ! are special characters in YAML. If you start a pattern with *, [, or !, you must enclose the pattern in quotes. ([link](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet))
How about just changing like below?
```
on: pull_request
```

Reply:
Good. I changed as you suggested